test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchGitHooksPersistenceIntegrationTest.java

```diff
@@ -0,0 +1,179 @@
+import org.kie.cloud.api.deployment.WorkbenchDeployment;
+import org.kie.cloud.api.scenario.ClusteredWorkbenchKieServerDatabasePersistentScenario;
+import org.kie.cloud.api.scenario.DeploymentScenario;
+import org.kie.cloud.api.scenario.WorkbenchKieServerPersistentScenario;
+import org.kie.cloud.common.provider.WorkbenchClientProvider;
+import org.kie.cloud.integrationtests.AbstractMethodIsolatedCloudIntegrationTest;
+import org.kie.wb.test.rest.client.WorkbenchClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@RunWith(Parameterized.class)
+public class WorkbenchGitHooksPersistenceIntegrationTest extends AbstractMethodIsolatedCloudIntegrationTest<DeploymentScenario> {
+
+    private static final Logger logger = LoggerFactory.getLogger(WorkbenchGitHooksPersistenceIntegrationTest.class);
+
+    @Parameterized.Parameter(value = 0)
+    public String testScenarioName;
+
+    @Parameterized.Parameter(value = 1)
+    public DeploymentScenario workbenchKieServerScenario;
+
+    private WorkbenchClient workbenchClient;
+
+    private WorkbenchDeployment workbenchDeployment;
+
+    private static final String GIT_HOOKS_REMOTE_DIR = "/opt/eap/standalone/data/kie/git/hooks";
+
+    @Parameterized.Parameters(name = "{0}")
+    public static Collection<Object[]> data() {
+        DeploymentScenarioBuilderFactory deploymentScenarioFactory = DeploymentScenarioBuilderFactoryLoader.getInstance();
+
+        WorkbenchKieServerPersistentScenario workbenchKieServerPersistentScenario = deploymentScenarioFactory.getWorkbenchKieServerPersistentScenarioBuilder()
+                .withGitHooksDir(GIT_HOOKS_REMOTE_DIR)
+                .build();
+
+        ClusteredWorkbenchKieServerDatabasePersistentScenario clusteredWorkbenchKieServerDatabasePersistentScenario = deploymentScenarioFactory.getClusteredWorkbenchKieServerDatabasePersistentScenarioBuilder()
+                .withGitHooksDir(GIT_HOOKS_REMOTE_DIR)
+                .build();
+
+        return Arrays.asList(new Object[][]{
+            {"Workbench + KIE Server - Persistent", workbenchKieServerPersistentScenario},
+            {"Clustered Workbench + KIE Server + Database - Persistent", clusteredWorkbenchKieServerDatabasePersistentScenario},});
+    }
+
+    @Override
+    protected DeploymentScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+        return workbenchKieServerScenario;
+    }
+
+    @Before
+    public void setUp() {
+        workbenchDeployment = deploymentScenario.getWorkbenchDeployments().get(0);
+        workbenchClient = WorkbenchClientProvider.getWorkbenchClient(workbenchDeployment);
+    }
+
+    @Test
+    public void testGitHooksPersistence() {
+        List<String> workbenchInstanceNames = workbenchDeployment.getInstances().stream().map(Instance::getName).collect(Collectors.toList());
+        if (workbenchInstanceNames.isEmpty()) {
+            throw new RuntimeException("Workbench was not correctly deploy.");
```

Comment:
Is this needed? AFAIK the scenario itself should check if Workbench is scaled when it is deployed.